sdk/core/core/src/az_span.c

```diff
@@ -44,43 +44,39 @@ AZ_NODISCARD az_span az_span_slice(az_span span, int32_t low_index, int32_t high
   return az_span_init(az_span_ptr(span) + low_index, high_index - low_index, capacity - low_index);
 }
 
-/**
- * ASCII lower case.
- */
 AZ_NODISCARD AZ_INLINE uint8_t az_ascii_lower(uint8_t value)
 {
   return 'A' <= value && value <= 'Z' ? value + AZ_ASCII_LOWER_DIF : value;
 }
 
-AZ_NODISCARD bool az_span_is_content_equal_ignoring_case(az_span a, az_span b)
+AZ_NODISCARD bool az_span_is_content_equal_ignoring_case(az_span span1, az_span span2)
```

Comment:
Is there any tooling we can leverage to make sure the source and header match exactly (including the parameter names)? Otherwise, there is nothing auditing things to make sure they don't go out of sync (other than manual inspection/validation in code review).

Reply:
Not that the C team is aware of; we've asked them about this a few months ago.